whois-api/src/main/java/net/ripe/db/whois/api/rest/RpslObjectStreamer.java

```diff
@@ -200,7 +202,11 @@ private void streamObject(@Nullable final RpslObject rpslObject) {
                 whoisObjectServerMapper.mapManagedAttributes(whoisObject, parameters, rpslObject);
                 whoisObjectServerMapper.mapResourceHolder(whoisObject, parameters, rpslObject);
 
-                streamingMarshal.writeArray(whoisObject);
+                if(streamingMarshal instanceof StreamingMarshalTextPlain){
```

Comment:
Add spaces for readability and consistent formatting (they're free! :) )

Reply:
Done, I have to take a look at the save actions to do this kind of things manually